package.json

```
@@ -2844,6 +2844,278 @@
+														"type": "command",
+														"command": "cmstead.jsRefactor.markAsAsync"
+													},
+													{
+														"key": "e",
+														"name": "export",
+														"type": "command",
+														"command": "cmstead.jsRefactor.addExport"
+													},
+													{
+														"key": "f",
+														"name": "introduce function",
+														"type": "command",
+														"command": "cmstead.jsRefactor.introduceFunction"
+													},
+													{
+														"key": "i",
+														"name": "inline variable",
+														"type": "command",
+														"command": "cmstead.jsRefactor.inlineVariable"
+													},
+													{
+														"key": "l",
+														"name": "lift and name function expression",
+														"type": "command",
+														"command": "cmstead.jsRefactor.liftAndNameFunctionExpression"
+													},
+													{
+														"key": "n",
+														"name": "negate expression",
+														"type": "command",
+														"command": "cmstead.jsRefactor.negateExpression"
+													},
+													{
+														"key": "p",
+														"name": "shift params",
+														"type": "command",
+														"command": "cmstead.jsRefactor.shiftParams"
+													},
+													{
+														"key": "r",
+														"name": "rename",
+														"type": "command",
+														"command": "cmstead.jsRefactor.rename"
+													},
+													{
+														"key": "s",
+														"name": "select from all refactorings",
+														"type": "command",
+														"command": "cmstead.jsRefactor.selectRefactoring"
+													},
+													{
+														"key": "c",
+														"name": "convert",
+														"type": "bindings",
+														"bindings": [
+															{
+																"key": ">",
+																"name": "convert function to arrow function",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToArrowFunction"
+															},
+															{
+																"key": "d",
+																"name": "convert function expression to function declaration",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToFunctionDeclaration"
+															},
+															{
+																"key": "e",
+																"name": "convert arrow function to function expression",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToFunctionExpression"
+															},
+															{
+																"key": "t",
+																"name": "convert string concat to template literal",
+																"type": "command",
+																"command": "cmstead.jsRefactor.convertToTemplateLiteral"
+															}
+														]
+													},
+													{
+														"key": "w",
+														"name": "wrap",
```

```suggestion
														"name": "+Wrap",
```

Discussion: should the name be surround like the vim?

---

I think I prefer surround myself now that you suggest it.